tests/TransactionHandler/handle.test.ts

```diff
@@ -0,0 +1,46 @@
+import { assert } from 'chai';
+import TransactionHandler from '../../src/TransactionHandler';
+import StakeRequestedHandler from '../../src/handlers/StakeRequestedHandler';
+import SpyAssert from '../SpyAssert';
+
+import sinon = require('sinon');
```

Comment:
This gives linting error with typescript 🔴 . 

Below works correctly 🍏 
import * as sinon from 'sinon';